services/user/AdminSys/ui/src/pages/dashboard-page.tsx

```diff
@@ -0,0 +1,10 @@
+export const DashboardPage = () => {
+    return (
+        <iframe
+            src="http://localhost:8091/d/psinode-dashboard?orgId=1&from=now-1h&to=now&refresh=5s&theme=light&kiosk=tv"
```

Comment:
How well is this going to work if we want to access the node through a reverse proxy?

Reply:
Great point.. it wouldn't... I think I could write a logic to check if the current address is `localhost`, if so I keep this address, otherwise I rewrite it to `/grafana/d/psinode-dashboard...` so we can instruct to add this proxy rewrite rule